app/models/measure_condition.rb

```diff
@@ -44,4 +45,8 @@ def is_weight_condition?
   def is_volume_condition?
     VOLUME_UNITS.include? condition_measurement_unit_code
   end
+
+  def is_eps_condition_code_v?
```

Comment:
```suggestion
  def is_eps_condition?
```

Technically v and eps are synonymous

Reply:
https://github.com/trade-tariff/trade-tariff-backend/blob/main/spec/factories/measure_factory.rb#L281-L285